app/src/main/java/com/github/se/cyrcle/ui/review/ReviewScreen.kt

```diff
@@ -146,6 +149,15 @@ fun ReviewScreen(
                               parking = selectedParking.uid,
                               rating = sliderToValue,
                               uid = reviewViewModel.getNewUid()))
+
+                      // Credit coins to user
+                      userViewModel.currentUser.value?.let { currentUser ->
```

Comment:
This piece of code is the same for when we add parking. 
We could create a function : userviewmodel.creditcoins(coins) and paste this code inside it